osm2pgsql.lua

```diff
@@ -499,128 +499,116 @@ function osm2pgsql.process_node(object)
         object.tags.wikidata == 'Q994730') then
         output_hstore['disputed_by'] = 'CN;RU;IN;GR'
     end
--- Recast Northern Cyprus as country label and turn off for several POVs including China and Russia
+
+-- Turn off Northern Cyprus label
     if object.tags.place and object.tags.wikidata == 'Q23681' then
-        output_hstore['place'] = 'country'
-        output_hstore['disputed_by'] = 'CN;RU;IN;GR;CY'
```

Comment:
I think it's OK to keep this... it's insurance that if we change Northern Cyprus to recognized by default later, these countries who have strong opinions will continue to object and get **kind:xx** = `unrecognzed`.